package.json

```diff
@@ -63,7 +63,7 @@
     "babel-plugin-dev-expression": "^0.2.2",
     "babel-plugin-module-resolver": "^3.1.1",
     "cross-env": "6.0.3",
```

Comment:
Since Yarn uses a portable shell you shouldn't need `cross-env` anymore
https://github.com/yarnpkg/berry/tree/2036672b8c3c51bf0a9ba2552a967c17fc95d6fe/packages/yarnpkg-shell

Reply:
good one, thanks!